pyproject.toml

```diff
@@ -25,6 +25,7 @@ mkdocs-material = "^9.1.21"
 [tool.poetry.group.dev.dependencies]
 connexion = "^2.14.2"
 pytest = "^7.4.0"
+pytest-cov = "^4.0.0"
 hypothesis = "^6.82.0"
 
```

Comment:
Can you try adding this here:
```toml
[tool.pytest]
adopt = --cov-report=xml # add remaining --cov flags
```

The goal is to remove the need to have those flags in the command line, making it implicit to generate code coverage reports.


Not 100% sure on the exact syntax so feel free to check pytest-cov documentation.

Reply:
modified the suggested command with stuff that I found on the internet.